src/algorithms/tracking/TrackParamTruthInit.h

```diff
@@ -26,6 +27,13 @@ namespace eicrecon {
         std::shared_ptr<spdlog::logger> m_log;
         TrackParamTruthInitConfig m_cfg;
         std::shared_ptr<TDatabasePDG> m_pdg_db;
+
+        double m_momentum_split{0.0};
```

Comment:
It is suggested to use Config structure for this class. See further.

Reply:
Out of scope.